### DISCUSSION TITLE
Door Sensors

### DISCUSSION THREAD

**Discussion by jayhabs** [tuya/tuya-homebridge#212] [Ideas]
I am trying to add 2 door sensors into homebridge.  I've followed the tutorials carefully and successfully added 2 lightstrips  but the sensors are not recognized .  Any thoughts.  Thanks

**Comment by 0x5e** [comment]
> I am trying to add 2 door sensors into homebridge.  I've followed the tutorials carefully and successfully added 2 lightstrips  but the sensors are not recognized .  Any thoughts.  Thanks 

You can try my fork version, some sensors supported.
